source/tests/pt/model/test_dpa3.py

```
@@ -0,0 +1,209 @@
+                repflow=repflow,
+                # kwargs for descriptor
+                exclude_types=[],
+                precision=prec,
+                use_econf_tebd=ect,
+                type_map=["O", "H"] if ect else None,
+                seed=GLOBAL_SEED,
+            ).to(env.DEVICE)
+
+            dd0.repflows.mean = torch.tensor(davg, dtype=dtype, device=env.DEVICE)
+            dd0.repflows.stddev = torch.tensor(dstd, dtype=dtype, device=env.DEVICE)
+            model = torch.jit.script(dd0)
```

Ruff (0.8.2)

Local variable `model` is assigned to but never used

Remove assignment to unused variable `model`

(F841)